azurelinuxagent/common/utils/networkutil.py

```diff
@@ -155,6 +185,18 @@ def __get_common_drop_command_params(wait, command, destination):
         cmd.extend(["conntrack", "--ctstate", "INVALID,NEW", "-j", "DROP"])
         return AddFirewallRules._add_wait(wait, cmd)
 
+    # This rule allows DNS TCP request to wireserver ip for non root users
+    @staticmethod
+    def get_iptables_accept_dns_tcp_request_command(wait, command, destination):
```

Comment:
From the code it seems the expectation of this function is to act as a common place for fetching iptable rules for different commands, not just append/accept. Could you please modify the name here to reflect that? Maybe make it a bit more generic for iptables?

Reply:
I have modified the name of function for just this rule as "get_firewall_accept_dns_tcp_rule" everywhere. Based on the input parameter it could be generated with "-A" or "-D" or "-I" or "-C"

in the next PR I will rename other rules as "get_firewall_root_accept_rule" ( or something better) and "get_firewall_drop_rule"